db/schema.rb

```diff
@@ -28,6 +28,17 @@
     t.index ["shelter_id"], name: "index_pets_on_shelter_id"
   end
 
+  create_table "reviews", force: :cascade do |t|
+    t.string "title"
+    t.integer "rating"
+    t.text "content"
+    t.string "image", default: "https://i.pinimg.com/originals/d1/21/f6/d121f64f59f7599486e3762fcd2f1475.jpg"
```

Comment:
sets a default value for the `image` attribute of a `review` in the `reviews` table